validate/validate.go

```diff
@@ -633,6 +634,20 @@ func (v *Validator) CheckLinuxResources() (msgs []string) {
 	logrus.Debugf("check linux resources")
 
 	r := v.spec.Linux.Resources
+
+	if r.CPU != nil {
+		if r.CPU.Cpus != "" {
+			if err := utils.UnitListValid(r.CPU.Cpus); err != nil {
+				msgs = append(msgs, err.Error())
```

Comment:
I'm not sure this check (or your following `mems` check) is grounded in the spec, but if the runtime-tools maintainer consensus is that it is, then the `validate/` changes from this PR belong in [v0.3.0][1].  I've also made a similar suggestion for #386 [here][2], in case runtime-tools maintainers want to handle both instances the same way.

[1]: https://github.com/opencontainers/runtime-tools/milestone/5
[2]: https://github.com/opencontainers/runtime-tools/pull/386#discussion_r141743231